### ISSUE TITLE
reproduce tutorial.ipynb

### ISSUE THREAD

**Issue by yuxiaokang-source** [ttgump/scDeepCluster#9]
I have noticed the issue https://github.com/ttgump/scDeepCluster/issues/6, so I have created the same environment as you mentioned in readme.md
<img width="751" alt="image" src="https://user-images.githubusercontent.com/59059267/166345485-d3795357-2bfe-4cac-96d6-633a15d9ba2b.png">
 After running the tutorial you provided in own machine,
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/59059267/166345577-7066a9be-b095-4195-b63a-89232b4609a1.png">
the ARI,NMI is very low and did't achieved perfect result. It seems strange that scDeepCluster is unstable for this simulation data,  And I noticed that the tutorial.ipynb you have running is not the version you provided in readme.md, the warning information in the tutorial.ipynb seems you are using python3.7, which is not compatible with the python 3.6.3 you mentioned in main page, can you give me some suggestions? Thank you!

**Comment by ttgump** [comment]
Hi, did you try the pytorch version? Old keras and tensorflow 1 indeed have the problem you mentioned, different version would generate different results. Thanks.

**Comment by ttgump** [comment]
You can generate datasets with larger dropout rates, set dropout.mid to a larger value such as 0.5 or 1 or 1.5 in the Splatter function. I mentioned in the methods part that we generated datasets with various dropout rates. You can use those settings as an example. Thanks.
